app/services/ethereum/bridge/service.go

```diff
@@ -48,10 +54,18 @@ func NewBridgeContractService(client *ethclient.EthereumClient, config config.Et
 		log.Fatalf("Failed to initialize Bridge Contract Instance at [%s]. Error [%s].", config.BridgeContractAddress, err)
 	}
 
-	return &BridgeContractService{
+	bridge := &BridgeContractService{
 		Client:           client,
 		contractInstance: contractInstance,
 	}
+
+	bridge.UpdateMembers(nil)
+	bridge.UpdateServiceFee(nil)
+
+	go bridge.listenForMemberUpdatedEvent(nil)
+	go bridge.listenForChangeFeeEvent(nil)
```

Comment:
Lets update the functions so that they don't receive `opts *bind.Callops`. We can pass `nil` inside the functions directly to the underlying `bsc.contractInstance` f.e:
`bsc.contractInstance.ServiceFee(nil)`
`bsc.contractInstance.MembersCount(nil)`